doc/user/install_guide.md

```diff
@@ -39,10 +41,12 @@ kubectl delete clusterrole etcd-operator
 kubectl delete clusterrolebinding etcd-operator
 ```
 
-## Installation via Helm
+## Installation using Helm
+
 **Disclaimer:** The following Helm chart is an external project not maintained by the etcd-operator maintainers; so it may not be up to date.
 
-etcd-operator is available as a [Helm
-chart](https://github.com/kubernetes/charts/tree/master/stable/etcd-operator).
-Follow the instructions on the chart to install etcd-operator on your cluster.
- 
+etcd-operator is available as a [Helm chart][etcd-helm]. Follow the instructions on the chart to install etcd-operator on clusters.
+
+
+[rbac-rules]: rbac.html
```

Comment:
rbac.html -> rbac.md

Reply:
I think you are correct with this one. I'll check to make sure it'll work once published.